analyze/workerd.ts

```diff
@@ -43,6 +44,31 @@ async function init(context: AnalyzeContext) {
         log.error(msg);
       },
     },
+    "arcjet:js-req/email-validator-overrides": {
+      isFreeEmail(domain) {
+        if (
+          [
+            "gmail.com",
+            "yahoo.com",
+            "hotmail.com",
+            "aol.com",
+            "hotmail.co.uk",
+          ].includes(domain)
```

Comment:
Do we need to check for `@`? 
```suggestion
          [
            "@gmail.com",
            "@yahoo.com",
            "@hotmail.com",
            "@aol.com",
            "@hotmail.co.uk",
          ].includes(domain)
```
Also, please move the email address array outside of the functions. It can be a static list at the top of the file.

Reply:
Nope, there's no need for the `@`, the domain that we're checking against is the one that's returned by the email parser rust lib.